packages/desktop-client/src/components/HelpMenu.tsx

```diff
@@ -100,14 +105,17 @@ export const HelpMenu = () => {
         <Menu
           onMenuSelect={item => {
             setMenuOpen(false);
-            handleItemSelect(item);
+            handleItemSelect(item as HelpMenuItem);
```

Comment:
If you change line 106 to have the correct type you should be able to infer it instead of asserting.
```
(item: HelpMenuItem)
```

Asserting types with ```as``` can lead to type issues

Reply:
Apologize if im not understanding correctly, but if i just change 
`` (item: item as HelpMenuItem)``
to 
``(item: HelpMenuItem)``
it gives me an error saying ``'HelpMenuItem' only refers to a type, but is being used as a value here.ts(2693)``

Reply:
Line 106

Change this line: 
```
          onMenuSelect={item => {
```

to:
```
          onMenuSelect={(item: HelpMenuItem) => {
```